Source/Core/VideoBackends/D3D12/D3DQueuedCommandList.cpp

```diff
@@ -0,0 +1,1207 @@
+		DEBUGCHECK(ColorRGBA[1] == 0.0f, "Error: Invalid assumption in ID3D12QueuedCommandList.");
+		DEBUGCHECK(ColorRGBA[2] == 0.0f, "Error: Invalid assumption in ID3D12QueuedCommandList.");
+		DEBUGCHECK(ColorRGBA[3] == 1.0f, "Error: Invalid assumption in ID3D12QueuedCommandList.");
+		DEBUGCHECK(pRects == nullptr, "Error: Invalid assumption in ID3D12QueuedCommandList.");
+		DEBUGCHECK(NumRects == 0, "Error: Invalid assumption in ID3D12QueuedCommandList.");
+
+		reinterpret_cast<D3DQueueItem*>(_queueArrayBack)->Type = D3DQueueItemType::ClearRenderTargetView;
+		reinterpret_cast<D3DQueueItem*>(_queueArrayBack)->ClearRenderTargetView.RenderTargetView = RenderTargetView;
+
+		_queueArrayBack += sizeof(ClearRenderTargetViewArguments) + sizeof(D3DQueueItemType) * 2;
+	}
+
+	void STDMETHODCALLTYPE ID3D12QueuedCommandList::ClearUnorderedAccessViewUint(
+		_In_  D3D12_GPU_DESCRIPTOR_HANDLE ViewGPUHandleInCurrentHeap,
+		_In_  D3D12_CPU_DESCRIPTOR_HANDLE ViewCPUHandle,
+		_In_  ID3D12Resource *pResource,
+		_In_  const UINT Values[4],
+		_In_  UINT NumRects,
+		_In_reads_opt_(NumRects)  const D3D12_RECT *pRects
+		)
+	{
+		// Function not implemented yet.
+		DEBUGCHECK(0, "Function not implemented yet.");
+	}
+
+	void STDMETHODCALLTYPE ID3D12QueuedCommandList::ClearUnorderedAccessViewFloat(
+		_In_  D3D12_GPU_DESCRIPTOR_HANDLE ViewGPUHandleInCurrentHeap,
+		_In_  D3D12_CPU_DESCRIPTOR_HANDLE ViewCPUHandle,
+		_In_  ID3D12Resource *pResource,
+		_In_  const FLOAT Values[4],
+		_In_  UINT NumRects,
+		_In_reads_opt_(NumRects)  const D3D12_RECT *pRects
+		)
+	{
+		// Function not implemented yet.
+		DEBUGCHECK(0, "Function not implemented yet.");
+	}
+
+	void STDMETHODCALLTYPE ID3D12QueuedCommandList::DiscardResource(
+		_In_  ID3D12Resource *pResource,
+		_In_opt_  const D3D12_DISCARD_REGION *pDesc
+		)
+	{
+		// Function not implemented yet.
+		DEBUGCHECK(0, "Function not implemented yet.");
+	}
+
+	void STDMETHODCALLTYPE ID3D12QueuedCommandList::SetMarker(
+		UINT Metadata,
+		_In_reads_bytes_opt_(Size)  const void *pData,
+		UINT Size
+		)
+	{
+		// Function not implemented yet.
+		DEBUGCHECK(0, "Function not implemented yet.");
+	}
+
+	void STDMETHODCALLTYPE ID3D12QueuedCommandList::BeginEvent(
+		UINT Metadata,
+		_In_reads_bytes_opt_(Size)  const void *pData,
+		UINT Size
+		)
+	{
+		// Function not implemented yet.
+		DEBUGCHECK(0, "Function not implemented yet.");
+	}
+
+	void STDMETHODCALLTYPE ID3D12QueuedCommandList::EndEvent()
+	{
+		// Function not implemented yet.
+		DEBUGCHECK(0, "Function not implemented yet.");
+	}
+
+	void STDMETHODCALLTYPE ID3D12QueuedCommandList::ExecuteIndirect(
+		_In_  ID3D12CommandSignature *pCommandSignature,
+		_In_  UINT MaxCommandCount,
+		_In_  ID3D12Resource *pArgumentBuffer,
+		_In_  UINT64 ArgumentBufferOffset,
+		_In_opt_  ID3D12Resource *pCountBuffer,
+		_In_  UINT64 CountBufferOffset
+		)
+	{
+		// Function not implemented yet.
+		DEBUGCHECK(0, "Function not implemented yet.");
+	}

```

Comment:
To save myself time, everything above this needs to have asterisks against the type.